repository/e.json

```diff
@@ -629,6 +624,18 @@
 				}
 			]
 		},
+		{
+			"name": "Elm Syntax Highlighting",
+			"details": "https://github.com/evancz/elm-syntax-highlighting",
+			"labels": ["elm", "language syntax", "highlighting"],
+			"previous_names": ["Elm Language Support"],
+			"releases": [
+				{
+					"sublime_text": "*",
```

Comment:
Since the syntax definition is only for ST3 (and in fact builds >=3092), please adjust this version specifier.

Reply:
Adjusted in https://github.com/wbond/package_control_channel/pull/7653/commits/d57f36b87cf9f5e8a993ff2d0a811e56a3616dc0